### DISCUSSION TITLE
Gesture support with the SDK

### DISCUSSION THREAD

**Discussion by sliptrixx** [elgatosf/elgato#72] [Ideas]
Adding gesture events for swiping left, right, up and down can help create interesting interactions. At least, an event for on touch up, or a stream of touch events so that the plugin can detect the gesture could be useful. 

I understand that swipe left and right are currently reserved to switching pages. However, it would be very useful to unlock the capabilities to all plugins to create mote interesting system.

**Comment by ChekTek** [comment]
Hi @sliptrixx,

I will relay this feature request to the Stream Deck team!

Best,
Zack
